axelrod/strategies/thuemorse.py

```diff
@@ -0,0 +1,60 @@
+from axelrod import Actions, Player
+
+class ThueMorse(Player):
+    """A player who cooperates or defects according to the Thue-Morse sequence."""
+
+    name = 'ThueMorse'
+    classifier = {
+        'memory_depth': float('Inf'),
+        'stochastic': False,
+        'makes_use_of': set(),
+        'inspects_source': False,
+        'manipulates_source': False,
+        'manipulates_state': False
+    }
+
+    def thuemorse_sequence(self, x):
+	    if x == 0:
+	        return 0
+	    if x%2==0:
+	        return self.thuemorse_sequence(x/2)
+	    if x%2==1:
+	        f = x-1
```

Comment:
If x is indeed the round, replace f with `previous_round` or similar (again: make the variables verbose: it helps readability).